esmvaltool/cmorizers/data/datasets.yml

```diff
@@ -0,0 +1,981 @@
+      Put all files in input_dir_path (no subdirectories with years).
+
+  ESACCI-CLOUD:
+    tier: 2
+    source: https://public.satproj.klima.dwd.de/data/ESA_Cloud_CCI/CLD_PRODUCTS/v3.0/
+    last_access: 2019-02-01
+    info: |
+      Download the data from:
+        L3C/AVHRR-PM/
+      To fill the gap 199409-199501, also download:
+        L3C/AVHRR-AM/AVHRR_NOAA-12/1994/ \
+          199409-ESACCI-L3C_CLOUD-CLD_PRODUCTS-AVHRR_NOAA-12-fv3.0.nc
+        L3C/AVHRR-AM/AVHRR_NOAA-12/1994/ \
+          199410-ESACCI-L3C_CLOUD-CLD_PRODUCTS-AVHRR_NOAA-12-fv3.0.nc
+        L3C/AVHRR-AM/AVHRR_NOAA-12/1994/ \
+          199411-ESACCI-L3C_CLOUD-CLD_PRODUCTS-AVHRR_NOAA-12-fv3.0.nc
+        L3C/AVHRR-AM/AVHRR_NOAA-12/1994/ \
+          199412-ESACCI-L3C_CLOUD-CLD_PRODUCTS-AVHRR_NOAA-12-fv3.0.nc
+        L3C/AVHRR-AM/AVHRR_NOAA-12/1995/ \
+          199501-ESACCI-L3C_CLOUD-CLD_PRODUCTS-AVHRR_NOAA-12-fv3.0.nc
+      Put all files under a single directory (no subdirectories with years).
+
+  ESACCI-FIRE:
+    tier: 2
+    source: ftp://anon-ftp.ceda.ac.uk/neodc/esacci/fire/data/
+    last_access: 2019-01-24
+    info: |
+      Download the data from:
+        burned_area/MERIS/grid/v4.1/
+      Put all files in input_dir_path (no subdirectories with years).
+
+  ESACCI-LANDCOVER:
+    tier: 2
+    source: ftp://anon-ftp.ceda.ac.uk/neodc/esacci/land_cover/data/land_cover_maps/
+    last_access: 2019-01-10
+    info: |
+      Download the 3 NetCDF files for 2000, 2005 and 2010.
+      Download the CCI-LC Tools from:
+        http://maps.elie.ucl.ac.be/CCI/viewer/download/lc-user-tools-3.14.zip
+      Unpack and run the CCI-LC Tools on each of the NetCDF files as follows:
+        bash lc-user-tools-3.14/bin/aggregate-map.sh \
+          -PgridName=GEOGRAPHIC_LAT_LON -PnumMajorityClasses=1 \
+          -PoutputAccuracy=false -PoutputPFTClasses=true \
+          -PoutputLCCSClasses=false -PnumRows=360 <inputfile.nc>
+      Put the resulting processed data in input_dir_path.
+
+      Caveat
+        The CCI-LC Tools must be applied before running this script.
+        The CCI-LC Tools require Java Version 7 or higher.
+        The input data are available for a single year and are copied over to
+        generate a time series over their time range of validity.
+
+  ESACCI-OC:
+    tier: 2
+    source: ftp://oceancolour.org/occci-v3.1/geographic/netcdf/monthly/chlor_a/
+    last_access: 2019-02-27
+    info: |
+      In case of issues with data download, check also the information provided at
+        OceanColour webpage https://esa-oceancolour-cci.org/
+      Put all files under a single directory (no subdirectories with years)
+      in ${RAWOBS}/Tier2/ESACCI-OC
+
+  ESACCI-OZONE:
+    tier: 2
+    source: ftp://anon-ftp.ceda.ac.uk/neodc/esacci/ozone/data/
+    last_access: 2019-02-01
+    info: |
+      Download the data from:
+        total_columns/l3/merged/v0100/
+      Put all files under a single directory (no subdirectories with years).
+
```

Comment:
I have just realized that downloading these raw files is not enough for the formatter script to work. One would also need additional files from: ftp://anon-ftp.ceda.ac.uk/neodc/esacci/ozone/data/limb_profiles/l3/merged/merged_monthly_zonal_mean/v0002/ as can be seen in the [cmorizer script](https://github.com/ESMValGroup/ESMValTool/blob/05f2241aa75a4ba8d0ee37f0b14ae4b1b368b987/esmvaltool/cmorizers/obs/cmorize_obs_esacci_ozone.ncl#L63). Could you please add them here and in the downloader script?

Reply:
Ok, it was a problem with the definition of the time coordinate